lib/wallet/api/http/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -2665,9 +2664,11 @@ constructTransaction api argGenChange knownPools poolStatus apiWalletId body = d
     withWorkerCtx api walletId liftE liftE $ \wrk -> do
         let db = wrk ^. dbLayer
             netLayer = wrk ^. networkLayer
+
+            trWorker :: Tracer IO W.WalletLog
```

Comment:
Unfortunately we need this again — and I don't know why.

Reply:
@Anviking I can empathise with your frustration!

In this particular case, at least we can say that:
- the type annotation serves as documentation
- it doesn't harm readability

With that in mind, I think we probably don't need to worry too much (for the moment).